repository/src/main/java/org/alfresco/transform/client/registry/CombinedConfig.java

```diff
@@ -47,58 +44,22 @@
 
 import java.io.IOException;
 import java.io.StringReader;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
-
-import static org.alfresco.repo.content.metadata.AsynchronousExtractor.isMetadataEmbedMimetype;
-import static org.alfresco.repo.content.metadata.AsynchronousExtractor.isMetadataExtractMimetype;
 
 /**
- * This class reads multiple T-Engine config and local files and registers them all with a registry as if they were all
+ * This class reads multiple T-Engine config and local files and registers as if they were all
  * in one file. Transform options are shared between all sources.<p>
  *
- * The caller should make calls to {@link #addRemoteConfig(List, String)} and {@link #addLocalConfig(String)} followed
- * by a call to {@link #register(TransformServiceRegistryImpl)}.
+ * The caller should make calls to {@link #addRemoteConfig(List, String)}, {@link #addLocalConfig(String)} or
+ * {@link #addTransformConfig(TransformConfig, String, String)} followed by a call to
+ * {@link #register(TransformServiceRegistryImpl)}.
  *
  * @author adavis
  */
-public class CombinedConfig
+public class CombinedConfig extends CombinedTransformConfig
 {
     private final Log log;
 
```

Comment:
The following code is now in the common CombinedTransformConfig class of the transform model for use by both the Repo and t-router.